src/components/ec-phone-number-input/ec-phone-number-input.vue

```diff
@@ -237,6 +238,10 @@ export default {
         return ['notification', 'modal', 'tooltip', 'level-1', 'level-2', 'level-3'].includes(value);
       },
     },
+    autocomplete: {
```

Comment:
We should add this controls into the stories, to let Chameleon user see that property right in the Storybook.